linkcheck/models.py

```diff
@@ -23,6 +23,7 @@
 except ImportError:
     USE_REVERSION = False
 
+from . import linkcheck_settings
```

Comment:
I had to use this syntax (in contrast to `from .linkcheck_settings import TOLERATE_BROKEN_ANCHOR` like for the other settings) to be able to change the value during tests - I found no easy way of modifying the settings file to make the settings automatically recognized as "Django settings" (which would make them available via `django.conf.settings` and enable e.g. the `@override_settings` decorator), do you know what the best practice for third party Django libraries is?

Reply:
Ah, thanks. Only tried to mock `linkcheck_settings.TOLERATE_BROKEN_ANCHOR` which didn't work because it's imported too soon :sweat_smile: